src/anim/anim.js

```diff
@@ -0,0 +1,755 @@
+Object.assign(pc, function () {
+
+    // animation interpolation type
+    var AnimInterpolation = {
```

Comment:
This isn't really consistent with other `pc` enums. I'd probably recommend:
* `pc.INTERPOLATION_STEP`
* `pc.INTERPOLATION_LINEAR`
* `pc.INTERPOLATION_CUBIC`

Reply:
Will do. I much prefer AnimInterpolation.STEP, but consistency is important!